MATLAB/orbit_controller/conversion.hpp

```diff
@@ -0,0 +1,53 @@
+#include "mex.hpp"
```

Comment:
This is exactly the same file shihao used for the attitude controller. I don't suppose its a good idea to have two copies of the same file, especially if you all plan on doing similar stuff for other gnc controllers. I can move it so that the attitude+orbit controller mex wrappers both include the same file. it might not be that big of a deal, idk what do you think?